### ISSUE TITLE
Content-Type for ca.crt prompts to install as system CA

### ISSUE THREAD

**Issue by jsha** [StreisandEffect/streisand#1441]
### Expected behavior:

Downloading ca.crt for OpenConnect or OpenVPN using an Android phone stores a file on the filesystem, and doesn't prompt.

### Actual Behavior:

Android prompts for the user's PIN, then prompts them to install the certificate. This is presumably due to the `Content-Type: application/x-x509-ca-cert` header.

Installing ca.crt as a system certificate is undesirable because it becomes trusted for HTTPS traffic, increasing the user's risk relative to installing ca.crt only in a VPN app.

### Steps to Reproduce:
1. Install streisand (I used the "localhost" method).
1. On an Android phone (tested on Pixel 2 running Android 9), visit /openconnect/, find the ca.crt link and tap it.

### Ansible Information

* Ansible version: 2.5.4
* Ansible system: Linux
* Host OS: Ubuntu
* Host OS version:  16.04
* Python interpreter: /usr/bin/python
* Python version: 2.7.12

### Streisand Information

* Streisand Git revision: 5a6139f819c1a20c1c72cf549e331173a707eb19
* Streisand Git clone has untracked changes: no
* Genesis role: localhost
* Custom SSH key: False

### Enabled Roles

* Shadowsocks enabled:  True
* Wireguard enabled: True
* OpenVPN enabled: True
* stunnel enabled: True
* Tor enabled: True
* Openconnect enabled: True
* TinyProxy enabled: True
* SSH forward user enabled: True
* Configured number of VPN clients: 5

**Comment by jsha** [comment]
Note: Probably the simplest way to fix this would be to store the CA certificate with a ".pem" extension.

While we're at it, I think it makes sense to store it as something like `ca-1.2.3.4.pem`, to better distinguish CA certificates from different Streisand instances.

**Comment by nopdotcom** [comment]
Yeah, that makes sense. Maybe if the machine is named `streisand`, we'll use the IP address? That might be useful logic elsewhere.
